.github/workflows/chart-validate.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+name: Charts Validate
+
+on:
+  pull_request:
+    branches: [main]
+    types: [opened, reopened, ready_for_review, synchronize] # added `ready_for_review` since draft is skipped
+    paths:
+      - helm-charts/**
```

Comment:
please add this line, so the test will be triggered in this PR. 
- .github/workflows/chart-validate.yaml

Reply:
I've added to the latest version. Please review again. Thank you.